### PR TITLE
Update data source revistion table to have null tx

### PR DESCRIPTION
**Pull request by taobun**

### CODE REVIEW

File: chain/db/table.go [hunk 1]
```diff
@@ -44,7 +44,7 @@ type DataSourceRevision struct {
 	Name           string `gorm:"not null"`
 	Timestamp      int64  `gorm:"not null"`
 	BlockHeight    int64  `gorm:"not null"`
-	TxHash         []byte `gorm:"not null"`
+	TxHash         []byte `sql:"default:null"`
```
Comment (sorawit) [inline chain/db/table.go:47]: I don't think we need to put in default?

**Review by sorawit — CHANGES_REQUESTED** [review]

File: chain/db/table.go (hunk 1 above)
Comment (sorawit) [reply chain/db/table.go:47]: NULL should be the automatic default for table that is nullable, no?

Comment (taobun) [reply chain/db/table.go:47]: If not set this tag, I cannot add nil of []byte as null in table.

**Review by sorawit — DISMISSED** [review]

**Review by evilpeach — DISMISSED** [review]

**Comment by taobun** [comment]
Something wrong when fix conflict new pr #997

**Pull request closed without merging**